config/routes.rb

```diff
@@ -1,56 +1,8 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+    post '/login', to: "sessions#create"
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+    resources :user
 
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+    root 'welcome#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.